dev-docs/source/ScriptRepository.rst

```diff
@@ -0,0 +1,87 @@
+.. _scriptrepository:
+
+=================
+Script Repository
+=================
+
+This page gives describes the implementation of the :ref:`script repository <WorkbenchScriptRepository>`
```

Comment:
```suggestion
This page describes the implementation of the :ref:`script repository <WorkbenchScriptRepository>`
```

Reply:
Nice catch! I've updated this now. Thanks.